cl/corpus_importer/management/commands/columbia_merge.py

```diff
@@ -0,0 +1,865 @@
+    if new_docket_source in VALID_UPDATED_DOCKET_SOURCES:
+        # Source is limited to those options because those are the only
+        # valid options when we sum the source with columbia source
+        docket.source = new_docket_source
+        docket.save()
+    else:
+        raise DocketSourceException(
+            f"Unexpected docket source: {new_docket_source}"
+        )
+
+
+def update_cluster_source(cluster: OpinionCluster) -> None:
+    """Update cluster source
+    :param cluster: cluster object
+    """
+    new_cluster_source = SOURCES.COLUMBIA_ARCHIVE + cluster.source
+    if new_cluster_source in VALID_MERGED_SOURCES:
+        cluster.source = new_cluster_source
+        cluster.save()
+    else:
+        raise ClusterSourceException(
+            f"Unexpected cluster source: {new_cluster_source}"
+        )
+
+
+def update_cluster_panel(
+    cluster: OpinionCluster,
+    panel_list: list[str],
+    panel_date: Optional[date] = None,
+) -> None:
+    """Update cluster's panel, this is done independently since it is a m2m relationship
+    :param cluster: the cluster object
+    :param panel_list: list with people names
+    :param panel_date: date used to find people
+    """
+
+    panel_list = [titlecase(p) for p in panel_list]
+
+    panel = async_to_sync(lookup_judges_by_last_name_list)(
+        panel_list, cluster.docket.court.id, panel_date
+    )
+
+    if panel:
+        cluster.panel.add(*Person.objects.filter(id__in=[p.id for p in panel]))
+
+
+def process_cluster(
+    cluster_id: int,
+    filepath: str,
+) -> None:
+    """Merge specified cluster id
+    :param cluster_id: Cluster object id to merge
+    :param filepath: specified path to xml file
+    """
+    columbia_data: dict = {}
+
+    cluster = (
+        OpinionCluster.objects.filter(id=cluster_id)
+        .exclude(
+            Q(source__in=VALID_MERGED_SOURCES)
+            | Q(docket__source__in=VALID_UPDATED_DOCKET_SOURCES)
+        )
+        .first()
+    )
+    if not cluster:
+        return
+
+    logger.info(msg=f"Merging {cluster_id} at {filepath}")
+    try:
+        soup = read_xml_to_soup(filepath)
+    except UnicodeDecodeError:
+        logger.warning(
+            f"UnicodeDecodeError: {filepath}, Cluster: {cluster_id}"
+        )
+        return
+
+    columbia_data["published"] = is_opinion_published(soup)
+
+    outer_opinion = soup.find("opinion")
+    if not outer_opinion:
+        # opinion wraps around all xml content
+        logger.warning("Ill formed xml columbia")
+        return
+
+    extracted_opinions = extract_opinions(outer_opinion)
+    columbia_data["opinions"] = prepare_opinions_found(extracted_opinions)
+    columbia_data["file"] = filepath
+    find_judge_and_type(columbia_data)
+    fix_simple_tags(soup, columbia_data)
+    extract_dates(columbia_data)
+
+    if not columbia_data["date_filed"]:
+        logger.warning(msg=f"No Date found {filepath}, Cluster: {cluster_id}")
+        return
```

Comment:
whats the point of this error.  we've already matched these opinions to CL - so its not the end of the world if you didnt find the date?  cant we get rid of this.